docs/lineage/manual-lineage-feature-guide.md

```diff
@@ -0,0 +1,123 @@
+import FeatureAvailability from '@site/src/components/FeatureAvailability';
+
+# About DataHub Manual Lineage
```

Comment:
Is there an existing Lineage feature guide we can add this into? I feel like we've been exploding "Lineage" as a feature into a bunch of different things... For exaple. when we add Time-Filtering, we should not add a new doc called "Filtering Lineage by Time". Instead, we should just add to a single "Lineage" feature guide..

Reply:
yeah that's a good point - right now we don't have a Lineage feature guide at all. we just have an intro page that simply points to a video about lineage (screenshot below)
<img width="1425" alt="image" src="https://user-images.githubusercontent.com/28656603/210891588-7ea53f76-7703-4bb0-933c-031161fc6d63.png">

Maybe this can be the beginning of a "Lineage" feature doc and I can mention that lineage can be ingested automatically at the beginning and link to the other page / video. But then obviously focus on manual lineage. That way the filtering by time can be included in here